images/ubuntu/templates/ubuntu-22.04.arm64.pkr.hcl

```diff
@@ -234,7 +270,7 @@ source "amazon-ebs" "build_image" {
   ami_virtualization_type                   = "hvm"
   ami_groups                                = var.aws_private_ami ? [] : ["all"]
   ebs_optimized                             = true
-  spot_instance_types                       = ["t4g.xlarge"]
+  spot_instance_types                       = ["m7g.xlarge"]
```

Comment:
Same as https://github.com/grafana/runner-images/pull/111#discussion_r1743891795
> This can be done on a follow up, or a precursor PR to this one actually, same for the [Fix merge conflicts preventing image creation](https://github.com/grafana/runner-images/pull/111/commits/d4a7daf34f3a0a02f711e72cf2d2d47817b0b764), do you think that would be best?

Reply:
Done in #115 and #116